### PR TITLE
Prometheus statistics improvements & additions.

### LINKED ISSUE #183: More Prometheus Statistics
* Attempts by (challenge|category)
* Solves by (challenge|category)
* Total points

### PR DESCRIPTION
**Pull request by jchristgit**
This makes changes to the existing solve and attempt count metrics to add
labels. This will likely break existing monitoring dashboards, but allows us to
be more granular in the solves (correct flag submissions) and attempts
(incorrect flag submissions) by challenge and category, whilst still allowing to
sum the total amount of flags via PromQL.

Caching of the values is removed. The values are moved to counters, which work
sanely in multiprocess mode in contrast to gauge mode. What we trade off for
this is the ability to decrease solve count, but after development discussions,
we have concluded that we currently do not really care about this use case.

A new metric for the total amount of points scored is introduced. As with the
updated metrics above, this is a counter. This does not allow us to directly run
queries such as retrieving the total amount of points at the end of the event -
we can use the database for that - instead, it will allow us to gather the
point increase in a given point of time, possibly over an entire event if the
range vector time selector is long enough (for instance
`increase(points_total[$__interval])` in a grafana dashboard).

Fixes #183.

### CODE REVIEW

**Comment by jchristgit** [comment]
codecov is wrong and I am right.

**Review by jerbob — APPROVED** [review]
Thanks Johannes!

File: entrypoints/backend.sh [hunk 1]
```diff
@@ -28,10 +28,10 @@ then
   /app/src/manage.py loaddata test_fixtures
 fi
 
-
+GUNICORN_COMMAND="gunicorn --chdir=src --reload --workers=12 --bind=0.0.0.0:8000 $@"
```
Comment (david-cooke) [inline entrypoints/backend.sh:31]: do we want 12 workers for our dev environment?

Comment (jchristgit) [reply entrypoints/backend.sh:31]: @jerbob broke this.

Comment (jchristgit) [reply entrypoints/backend.sh:31]: No, we do not.

**Review by Bentechy66 — APPROVED** [review]

**Pull request merged**